episodes/itksnap.Rmd

```diff
@@ -0,0 +1,630 @@
+
+### Opening and Viewing DICOM images
+
+* Open DICOM images  
+
+Open ITK-SNAP application and the directory where you see "3.000000-CT Lung  3.0  B31f-61322" where you see `1-*.dcm` files, drag directory and drop it into ITK application,  select File format to `DICOM Image Series`, and clid `Next` and `Finish`
+![ITK-SNAP main window](../fig/itk-snap-viewing-dicoms.svg)
+
+### Navigating DICOM images
+
+Here, we provide instructions for handling multiple DICOM images, applying overlays, viewing image information, and using color maps in ITK-SNAP.
+
+#### 1. Loading Multiple DICOM Images as Separate Images
+
+1. **Open the First DICOM Series**:
+   - Follow the steps above to load and view the first DICOM series.
+
+2. **Open the Second DICOM image in a New Window**:
+   - Go to `File` > `New itksnap Window`.
+   - Select `Open DICOM Series` and choose the second DICOM series.
+   - This will open the second series in a separate ITK-SNAP window.
+   - This can be repeated for multiple DICOM series.
+
+![ITK-SNAP new window](../fig/new-itk-snap-window.png)
+
+#### 2. Applying Overlays
+
+1. **Load Primary Image**:
+   - Open your primary DICOM series as described above.
+
+2. **Load Overlay Image**:
+   - Go to `File` > `Add Another Image`.
+   - Select the image you want to use as an overlay. This could be another DICOM series or a previously saved segmentation.
+
+![ITK-SNAP additional image](../fig/open-additional-image.png)
+
+![ITK-SNAP overlay](../fig/overlay-wizard.png)
+
+3. **Adjust Overlay Settings**:
+   - Go to the `Layers` panel.
+   - Right click on the second DICOM Series and adjust the opacity and blending settings to properly visualise the overlay on the primary image.
+
+![ITK-SNAP layers panel](../fig/layers-panel.png)
+
+
+#### 4. Viewing Image Information
+
+1. **Access Image Information**:
+   - Go to `Tools` > `Image Information`.
+   - A dialog box will appear displaying metadata and other relevant information about the loaded DICOM images.
+
+![ITK-SNAP additional image](../fig/image-info.png)
+
+
+
+#### 5. Applying Color Maps
+
+1. **Load the Image**:
+   - Open the desired DICOM series.
+
+2. **Open Color Map Settings**:
+   - Go to `Tools` > `Color Maps Editor`.
+
+3. **Apply and Adjust Color Map**:
+   - Choose a predefined color map from the list or create a custom one.
+   - Adjust the intensity and transparency settings as needed to enhance the visualisation of the images.
+
+![ITK-SNAP additional image](../fig/colour-map-editor.png)
+
+
+### Converting DICOM images to NifTi
+
+#### Using ITK-SNAP
+Navigate the ITK-snap by clicking 'File' --> 'Save image' --> rename image and choose format as 'NiFTI'.
+
+#### Using [dicom2nifti](https://github.com/icometrix/dicom2nifti)
+```
+from pathlib import Path 
+import dicom2nifti 
+dicoms = Path("~/3_000000-CT_Lung_30_B31f-61322")
+dicom2nifti.convert_directory(dicoms, ".", compression=True, reorient=True)
+```
+
```

Comment:
Good suggestion but we have 30 to 40 python snippets that would overflow episode, so let's keep it as it is to avoid user making too many clicks to expand collapse callouts